codewof/static/js/question_types/base.js

```diff
@@ -25,6 +28,31 @@ function clear_submission_feedback() {
     $('#submission_feedback').empty();
 }
 
+function update_gamification(data) {
+    curr_points = data.curr_points;
+    $('#user_points_navbar').innerText = curr_points;
+    $("#user_points_navbar").load(location.href + " #user_points_navbar"); // Add space between URL and selector.
+
+    point_diff = parseInt(data.point_diff);
+    if(point_diff > 0) {
+        $("#point_toast_header").text("Earned " + point_diff.toString() + " points!");
+        $("#point_toast_body").text("Your total points are now " + data.curr_points.toString() +"!");
+        $(document).ready(function(){
+            $("#point_toast").toast('show');
+        });
```

Comment:
On another note this is pretty confusing to the reader when they only have a second to look at it. It's easy to miss the smaller text `Earned X points!` and misinterpret the larger message `Your total points are now Y!` as the number of points you just earned (X). I think the running total can be left out altogether since a copy is only a few pixels away anyways

In fact, make them last longer also, they're too quick to read unless you're expecting them

Reply:
Made them appear for 3s but I think it might not be working when 2 toasts appear at the same time. Haven't looked into it yet, making note here so I don't forget.